packages/server-wallet/e2e-testing/test-data/load-data.json

```diff
@@ -1,9 +1,9 @@
 [
  {
   "type": "CreateChannel",
-  "jobId": "quaint-ancient-calm-crowd",
+  "jobId": "cuddly-raspy-whining-pillow",
```

Comment:
This file should be ignored by git

Reply:
This is more to show a sample of load data and provide a default for the start all script. The load generator generates files in `temp` by default.

Reply:
I see. It’s a pretty large diff on a decently large file, but if it’s not going to change much, that seems fine